bundles/binding/org.openhab.binding.fs20/src/main/java/org/openhab/binding/fs20/internal/FS20Binding.java

```diff
@@ -149,23 +149,30 @@ public void updated(Dictionary<String, ?> config)
 		if (config != null) {
 
 			Boolean configChanged = false;
+			/**
+			 * Valid values of the baudRateString
+			 * "75", "110", "300", "1200", "2400", "4800", 
+			 * "9600", "19200", "38400", "57600", "115200"
+			 * @see org.openhab.io.transport.cul.internal.CULSerialHandlerImpl
+			 */
 			String baudRateString = (String) config.get(KEY_BAUD_RATE);
 			if(StringUtils.isNotBlank(baudRateString)){
-				properties.put(KEY_BAUD_RATE, Integer.parseInt(baudRateString));
+				properties.put(KEY_BAUD_RATE, baudRateString);
 				configChanged = true;
 			}
```

Comment:
since the valid baudRates are known couldn't we implement a validator in io.transport.serial so other serial based bundles could benefit as well?